packages/frontend/src/page/components/opening-countdown/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { ReactElement, useEffect, useState } from "react";
```

Comment:
Wouldn't it be better to update the UI `Timer` component to be styleable like we need?

Reply:
It could be a nice improvement, we may want to refactor this in the future.